ApiBundle/Facade/PaginateCollectionFacade.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Facade;
+
+use JMS\Serializer\Annotation as Serializer;
+use OpenOrchestra\BaseApi\Facade\AbstractFacade;
+
+/**
+ * Class PaginateCollectionFacade
+ */
+abstract class PaginateCollectionFacade extends AbstractFacade
+{
+    /**
+     * @Serializer\Type("integer")
+     * @Serializer\SerializedName("recordsTotal")
+     */
+    public $recordsTotal;
+
+    /**
+     * @Serializer\Type("integer")
+     * @Serializer\SerializedName("recordsFiltered")
+     */
+    public $recordsFiltered;
+
+    /**
+     * @param $recordsTotal
+     */
+    public function setRecordsTotal($recordsTotal)
```

Comment:
as the properties are public and they are integer, you do not need a setter to set them